registry/storage/driver/s3-aws/s3.go

```diff
@@ -269,6 +295,10 @@ func FromParameters(parameters map[string]interface{}) (*Driver, error) {
 // New constructs a new Driver with the given AWS credentials, region, encryption flag, and
 // bucketName
 func New(params DriverParameters) (*Driver, error) {
+	if !params.V4Auth && params.Region == "eu-central-1" {
```

Comment:
This is actually the case for `eu-central-1`, `ap-south-1`, and `ap-northeast-2`. (See http://docs.aws.amazon.com/general/latest/gr/rande.html#s3_region)